docs/content/latest/api/ysql/datatypes/type_array/functions-operators/any-all.md

```diff
@@ -0,0 +1,306 @@
+---
+title: ANY and ALL—test if an element is in an array
+linkTitle: =ANY and =ALL
+headerTitle: =ANY and =ALL — test if an element is in an array
+description: The =ANY and =ALL operators test if an element is in an array
+menu:
+  latest:
+    identifier: any-all
+    parent: array-functions-operators
+    weight: 10
+isTocNested: true
+showAsideToc: true
+---
+
+## Overview
+
+**Purpose:** Return `TRUE`, `FALSE`, or `NULL` according to the outcome of the particular comparison, `=ANY` or `=ALL`, between the input [LHS](https://en.wikipedia.org/wiki/Sides_of_an_equation) _element_ and input [RHS](https://en.wikipedia.org/wiki/Sides_of_an_equation) array. Each of the LHS and the RHS is, in general, an expression. Further, the RHS expression must be parenthesized.
+
+The value to which the LHS evaluates is compared, using the appropriate data type overload of the `=` operator, to each of the RHS expression's evaluated _elements_ to yield a `boolean` result (`TRUE`, `FALSE`, or `NULL`) according to the normal rules for _element_ comparison. (See the [Semantics demonstration](./#semantics-demonstration) section.)
```

Comment:
```suggestion
The value to which the LHS evaluates is compared, using the appropriate data type overload of the `=` operator, to each of the RHS expression's evaluated _elements_ to yield a `boolean` result (`TRUE`, `FALSE`, or `NULL`) according to the normal rules for _element_ comparison. (See [Semantics demonstration](./#semantics-demonstration).)
```

Reply:
I did a global search for `the [` and `) section and changed them all as you want. See that paragraph that used to start like this:

> The [Equality and inequality semantics](./#equality-and-inequality-semantics) section demonstrates each of the rules that this _"Comparison operators overview"_  section has stated.

It used to read well because "section" is a noun that serves well as the subject of a sentence. Now it reads badly because "Equality and inequality semantics" simply doesn't serve this purpose. But... it's what you style guide robotically insist on.

Now the doc reads like Russian where they've never heard of the definite article.